ddtrace/contrib/cassandra/session.py

```diff
@@ -46,7 +45,7 @@ def traced_execute(func, instance, args, kwargs):
     query = kwargs.get("kwargs") or args[0]
 
     with tracer.trace("cassandra.query", service=service, span_type=cassx.TYPE) as span:
-        span.resource = _sanitize_query(query)
+        span = _sanitize_query(span, query)
```

Comment:
`_sanitize_query` both alter and return `span`. 
Could we just do `_sanitize_query(span, query)` and return nothing? At least that would make it clear.